docs/Operators.md

```diff
@@ -5637,6 +5686,55 @@ This version of the operator has been available since version 1 of the default O
 </dl>
 
 
+#### Examples
+
+<details>
+<summary>reducel2</summary>
+
+```python
+
+data = np.array(
+    [[[1, 2], [3, 4]], [[5, 6], [7, 8]], [[9, 10], [11, 12]]],
+    dtype=np.float32)
+
+node = onnx.helper.make_node(
+    'ReduceL2',
+    inputs=['data'],
+    outputs=['reduced'],
+    axes=[2],
+    keepdims=0
+)
+
+reduced = np.array([
+    [2.23606777, 5.],
```

Comment:
Since the output precision matters in this case, could you use the numpy function to calculate the results. You can add the output value in the comment. The users can still quickly check them. :-)

BTW, I think sqrt(5) is closer to 2.2360679775, 2.23606777 is probably not precise enough.